docs/submit-opentargets-batch.md

```diff
@@ -14,7 +14,7 @@ export OT_RELEASE=YYYY-MM
 
 # Year and month of ClinVar release used (see step 1.1 Download ClinVar data).
 # Note that this is *different* from the OpenTargets release year/month.
-export CLINVAR_RELEASE=YYYY_MM
+export CLINVAR_RELEASE=YYYY-MM
```

Comment:
I thought there was a step that needed the `_`. We'll see if the `-` works in all steps.

Reply:
It was used only once in the previous versions of the protocol, to infer the name of the file which OT provide back, with the consequence & gene mappings. Its actual name for batch 2019-09 was `mergeEVA_uniq_clinvar_2019-04_19_09_manually_corrected.out.gz`, so I assumed it would always be `mergeEVA_uniq_clinvar_${OT_BATCH}_${CLINVAR_BATCH}_manually_corrected.out.gz`, where `$OT_BATCH` uses a hyphen (as usual) and `$CLINVAR_BATCH` uses an underscore. However the actual name pattern of the file provided back by OT varies from batch to batch, so I removed that, and the protocol now uses direct assignment of `$OT_OUTPUT_FILE`